src/defaultConfig.ts

```diff
@@ -87,5 +88,6 @@ export const defaultSecurityConfig = (serverlUrl: string): Partial<ModuleOptions
     hashScripts: true,
     hashStyles: false
   },
-  sri: true
+  sri: true,
+  //runtimeHooks: true
```

Comment:
We can remove it as it doesn't exist anymore.
I left it as a reminder that it used to exist.
`runtimeHooks` are now always enabled, because everything goes through the unified context

Reply:
I'm taking it out